examples/with-plugins/plugins/my-plugin.ts

```diff
@@ -0,0 +1,9 @@
+import { defineNuxtPlugin } from '#imports'
```

Comment:
This is not required. We might comment it out for readability or remove.

```suggestion
// import { defineNuxtPlugin } from '#imports'
```